src/identity/components/GlobalHeader/GlobalHeaderDropdown/GlobalHeaderTypeAheadMenu.tsx

```diff
@@ -57,8 +66,8 @@ export class GlobalHeaderTypeAheadMenu extends React.Component<Props, State> {
     }
   }
 
-  private handleInputChange = (event: ChangeEvent<HTMLInputElement>) => {
-    const filterString = event?.target?.value
+  private handleInputChange = (e: ChangeEvent<HTMLInputElement>) => {
```

Comment:
same thing here for changing `event` to `e`

Reply:
per discussion - will change these to `evt`